### ISSUE TITLE
About improvement of IForest

### ISSUE THREAD

**Issue by letter1225** [selimfirat/iforest-variants#1]
Hello, have you read about the paper **Improving iForest with Relative Mass**? It's an interesting improvement of iForest to endow it with the capability to detect local outliers.  Will you consider adding that in your project? https://link.springer.com/chapter/10.1007/978-3-319-06605-9_42

**Comment by changyunke** [comment]
yes, i also think which issue should be solved in the project of iforest
